lib/client-assets.php

```diff
@@ -566,7 +566,13 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	);
 
 	// Initialize the editor.
-	wp_add_inline_script( 'wp-editor', 'wp.api.init().done( function() { wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost ); } );' );
+	$editor_settings = array(
+		'wideImages' => get_theme_support( 'wide-images' ),
+	);
+	wp_add_inline_script( 'wp-editor', 'wp.api.init().done( function() {'
+		. 'wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost, ' . json_encode( $editor_settings ) . ' ); '
```

Comment:
I think there's a global reflection to have here the more we create JS UI. (not only Gutenberg). 
A proposal could be:

-  if you need `date` and you enqueue `wp-date` you automatically get a `wpDateSettings` global variable.
- if you need `i18n` and you enqueue `wp-i18n` you automatically get a `wpI18nSettings` global.
- if you need `themes support` and you enqueue `wp-themes` you get `wpThemeSettings` global.

I don't if this proposal is good but we need to find a consistent way of retrieving these settings. An automatically injected global variable or an endpoint (the first one is more "reactive" and performant I guess)

But at the same time, I'd like us to avoid the globals in the Editor Code itself (to make it reusable), so a `settings` variable in `createEditorInstance` makes sense to me and could be computed using these globals.